acceptance/tests/custom_facts/error_in_custom_file_doesn_not_affect_api.rb

```diff
@@ -0,0 +1,117 @@
+test_name 'Facter api works when there is an error inside a custom fact file' do
+  tag 'risk:high'
+
+  first_file_content = <<-EOM
+    Facter.add(:custom_fact_1) do
+      setcode do
+          'custom_fact_1_value'
+      end
+    end
+     
+    # some error
+    nill.size
+     
+    Facter.add(:custom_fact_2) do
+      setcode do
+          'custom_fact_2_value'
+      end
+    end
+     
+    Facter.add(:custom_fact_3) do
+      setcode do
+          'custom_fact_3_value'
+      end
+    end
+  EOM
+
+  second_file_content = <<~EOM
+    Facter.add(:custom_fact_4) do
+      setcode do
+          'custom_fact_4_value'
```

Comment:
The strings in all the custom facts are not indented correctly.

Reply:
Done